corehq/apps/locations/models.py

```diff
@@ -71,12 +77,31 @@ class LocationType(models.Model):
     shares_cases = models.BooleanField(default=False)
     view_descendants = models.BooleanField(default=False)
 
+    emergency_level = StockLevelField(default=0.5)
+    understock_threshold = StockLevelField(default=1.5)
+    overstock_threshold = StockLevelField(default=3.0)
+
     objects = LocationTypeManager()
 
+    def _populate_stock_levels(self):
+        from corehq.apps.commtrack.models import CommtrackConfig
+        ct_config = CommtrackConfig.for_domain(self.domain)
+        if (
+            (ct_config is None)
+            or (not Domain.get_by_name(self.domain).commtrack_enabled)
+            or LOCATION_TYPE_STOCK_RATES.enabled(self.domain)
+        ):
+            return
+        config = ct_config.stock_levels_config
+        self.emergency_level = config.emergency_level
+        self.understock_threshold = config.understock_threshold
+        self.overstock_threshold = config.overstock_threshold
+
     def save(self, *args, **kwargs):
         if not self.code:
             from corehq.apps.commtrack.util import unicode_slug
             self.code = unicode_slug(self.name)
+        self._populate_stock_levels()
```

Comment:
Yeah, not that I can think of offhand.  Saving LocationTypes is a pretty infrequent operation though (except of course for this migration, where we save ALL of them), so I'm not too concerned about the performance of it.